### PR TITLE
Described ibexa:content:remove-duplicate-fields in the 4.6.13 update doc

### PR DESCRIPTION
**Pull request by mnocon**
Target: all versions.

Adjusted documentation after the 4.6.13 release.

The only thing to mention is the command introduced in https://github.com/ezsystems/ezplatform-kernel/pull/410.

Preview: [preview](https://ez-systems-developer-documentation--2530.com.readthedocs.build/en/2530/update_and_migration/from_4.6/update_from_4.6/#v4613)

### CODE REVIEW

File: docs/update_and_migration/from_4.6/update_from_4.6.md [hunk 1]
```diff
@@ -138,8 +138,27 @@ merge with your custom settings if needed, and commit them to Git.
...
 
 ```php
     Ibexa\Bundle\CoreSearch\IbexaCoreSearchBundle::class => ['all' => true],
 ```
+
+## v4.6.13
+
+This release comes with a command to clean up the duplicated entries in the `ezcontentobject_attribute` table, caused by the issue described in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562).
+
+If you're affected you can remove the duplicated entries by running the `ibexa:content:remove-duplicate-fields` command:
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_4.6/update_from_4.6.md:151]:
```suggestion
If you're affected you can remove the duplicated entries by running the following command:
```
I think that it's not needed to repeat it.

File: docs/update_and_migration/from_4.6/update_from_4.6.md [hunk 2]
```diff
@@ -138,8 +138,27 @@ merge with your custom settings if needed, and commit them to Git.
...
+php bin/console ibexa:content:remove-duplicate-fields
+```
+
+!!! caution
+
+    Remember about [**proper database backup**](backup.md) before running the command in the production environment.
+
+You can customize the behavior of the command with the following options:
+
+- `batch-size` or `b` - number of attributes affected per iteration. Default: 10000.
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_4.6/update_from_4.6.md:162]:
```suggestion
- `batch-size` or `b` - number of attributes affected per iteration. Default value = 10000.
```
To make it consistent with existing docs (example: https://doc.ibexa.co/projects/userguide/en/latest/getting_started/dashboard/dashboard_block_reference/).

File: docs/update_and_migration/from_4.6/update_from_4.6.md [hunk 3]
```diff
@@ -138,8 +138,27 @@ merge with your custom settings if needed, and commit them to Git.
...
+- `max-iterations` or `i` - max. iterations count (default or -1: unlimited). Default: -1.
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_4.6/update_from_4.6.md:163]:
```suggestion
- `max-iterations` or `i` - max. iterations count (default or -1: unlimited). Default value = -1.
```

File: docs/update_and_migration/from_4.6/update_from_4.6.md [hunk 4]
```diff
@@ -138,8 +138,27 @@ merge with your custom settings if needed, and commit them to Git.
...
+- `sleep` or `s` - wait time between iterations, in milliseconds. Default: 0.
```
Comment (julitafalcondusza) [inline docs/update_and_migration/from_4.6/update_from_4.6.md:164]:
```suggestion
- `sleep` or `s` - wait time between iterations, in milliseconds. Default value = 0.
```

**Review by julitafalcondusza — APPROVED** [review]
See my comments.

**Review by tbialcz — APPROVED** [review]

**Pull request merged**